src/Datadog.Trace/Tags.cs

```diff
@@ -175,5 +175,11 @@ public static class Tags
         /// Configures Trace Analytics.
         /// </summary>
         public const string Analytics = "_dd1.sr.eausr";
+
+        /// <summary>
+        /// Language tag
+        /// RFC: https://github.com/DataDog/architecture/blob/master/rfcs/apm/integrations/runtime-metrics-collection/rfc.md#traces
```

Comment:
Could you remove the RFC link since it is not publicly accessible?

Reply:
👍